### ISSUE TITLE
caldav sync support

### ISSUE THREAD

**Issue by mr-gosh** [federicoiosue/Omni-Notes#324]
Did you consider syncability via caldav and or owncloud?

**Comment by federicoiosue** [comment]
Not directly into the app. 
The autobackup branch performs JSON automatic backup that could be synchronized with third party applications anyway

**Comment by aproposnix** [comment]
Hi I just found your app today on F-Droid. It's really fantastic.  

Having the ability to sync with nextcloud (or Owncloud) would be very beneficial.  Nextcloud developers could then develop an App for working with the data  - it is in my opinion the missing link for those seeking an Evernote replacement.

**Comment by nonbinary** [comment]
+1 for the nextcloud integration

**Comment by mr-gosh** [comment]
the integration could work like the following:
The standard to be used is [RFC5545](https://tools.ietf.org/html/rfc5545#section-3.6.2) the VTODO part and/or the VJOURNAL parat of the spec.

any constrain like the deadline, priority and so on should be based on the RFC the content should be in some kind of markup language in the Main Field and the embedded iamges and so on should be Attachements to the cal entry.

**Comment by blausand** [comment]
**+1** for going with the standards, @mr-gosh!
There are 999 Note Apps and softwares out there, and the ones going with standards lack the **feature completeness _Omni-Notes_** offers.
It's worth a million if this can be resolved. But the basic **file format is crucial**!
I think **HTML** is most promising as:

1. **HTML** can embed pictures (Base64)
2. **HTML** can embed [microformats](https://en.wikipedia.org/wiki/Microformat) for Calenders and Contacts
3. **HTML** can be edited in plain text easily; simple editors can start events and contacts using simple HTML templates. 

Just my 50ct.

**Comment by mr-gosh** [comment]
HTML is still not as simple as markdown - which also can be edited in any plaintext editor...

**Comment by Svadkos** [comment]
I think that markdown is better too. It's simpler to edit and can be converted in html easily, if needed.

**Comment by aproposnix** [comment]
I disagree with markdown. There are a million and one other md note taking applications.

**Comment by blausand** [comment]
I really gave md a try and i know about it's advantages which is why i sort of like it's brave attitude.
For the reasons stated above (mainly feature coverage and file open handlers), HTML faces a way brighter future.
Again, just my 50ct.

**Comment by nonbinary** [comment]
The purpose of md is to have formatted text that is still readable.  
HTML is ugly if you view the code.  
Since that's the purposes of the markup languages, I'd say that that's what should decide what to user here.  
Do we want users to view the raw text? Then go with markdown. If not, go with HTML.

Apart from the purpose, it's just a matter of taste, so this discussion threatens to overshadow the main issue: caldav sync.

**Comment by mr-gosh** [comment]
It's getting off-topic
Still the main question is "what about caldav syncable format"?

**Comment by Jazcafe** [comment]
+1 for CalDAV

**Comment by dapgo** [comment]
+1 CalDav (VJOURNAL) +1 MD Mark Down

**Comment by gjelsas** [comment]
+1 for calDAV and thereby Nextcloud support!
